src/renderer/components/AircraftSection/index.tsx

```diff
@@ -203,6 +206,21 @@ const index: React.FC<Props> = (props: Props) => {
         }
     }
 
+    function getLastInstalledBranch(): string {
```

Comment:
**nitpick:** this function can be extracted from the component and put in a separate folder that deals with download logic. I recommend something like `common/download.utils.ts`